platform/i18n/src/locales/en-US/DatePicker.json

```diff
@@ -1,5 +1,9 @@
 {
   "Clear dates": "Clear dates",
+  "Close": "$t(Common:Close)",
```

Comment:
Should we add this to every DatePicker.json file?

Reply:
I think that test-LNG and en locales are enough. In some localizations there is either no "Close" key or a Common.json file.
So I would like to do test-LNG and en first and then I can make a template for other languages. What do you think?

Reply:
Yes. That makes sense.